### PR TITLE
Contact form

### LINKED ISSUE #15: Create a contact form in another page
Have a new page on the website which contains a contact form. 
Include and work with various field types after reading material from mdn.

The form should not work. No JavaScript requires at this point.
Only look and feel

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
#15

# PR 23 - 12 - 23

## Changes

- Adjusted name on the picture folders
- Add contact-us form and link that on the footer section on the home page

Screenshot 2023-12-23 102640

Screenshot 2023-12-23 102659

### CODE REVIEW

File: contact-us.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="CSS/contact-us style/form.css">
+    <title>Contact us</title>
+</head>
+<body>
+    <div class="form__container">
+        <form>
```
Comment (vladiiancu) [inline contact-us.html]: A form should have an id generally speaking for easy recognition

File: home.html [hunk 2]
```diff
@@ -12,12 +12,12 @@
...
     <div class="header">
         <div class="header__logoIcon">
-            <img src="/Assets/images/home/company_logo.svg" alt="Company Logo">
+            <img src="/Assets/images/home logo/company_logo.svg" alt="Company Logo">
             <h1 class="header__logoTitle">Pet Care</h1>
         </div>
 
         <div class="sandwich_menu">
-            <img src="/Assets/images/home/cat-logo.svg" alt="">
+            <img src="/Assets/images/home logo/cat-logo.svg" alt="">
```
Comment (vladiiancu) [inline home.html:20]: These changes are not really related to the issue and PR, right?

Comment (Alexandru-Bulai) [reply home.html:20]: They are not indeed, i should have made another branch for that change

File: contact-us.html (hunk 1 above)
Comment (Alexandru-Bulai) [reply contact-us.html]: i see, i will fix that

File: CSS/contact-us style/form.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'poppins', sans-serif;
```
Comment (vladiiancu) [inline CSS/contact-us style/form.css]: Same situation as the comment from another PR

**Review by vladiiancu — CHANGES_REQUESTED** [review]
Changes have to be made also in order to resolve conflicts

**Comment by Alexandru-Bulai** [comment]
I updated the whole structure because i wasnt happy with the looking of the other one.
Now should be looking a little more professional.

Screenshot 2023-12-26 212523

Screenshot 2023-12-26 212300

Screenshot 2023-12-26 212311

File: CSS/contact-us style/form.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+}
+
+label {
+    font-size: 1.5rem;
+    padding: 0.5rem 0;
+    cursor: text;
+}
+
+input, textarea {
+    width: 100%;
```
Comment (vladiiancu) [inline CSS/contact-us style/form.css:68]: Around line 30 the input and textarea fields are already targeted and assigned some values. 

Take that into consideration

File: contact-us.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            </div>
+            <div class="form__inputGroup">
+                <input type="email" id="email" placeholder="Email" required>
+            </div>
+            <div class="form__inputGroup">
+                <textarea id="message" rows="4" placeholder="Your Message"></textarea>
+            </div>
+            <div id="buttons">
+                <button type="submit">SUBMIT</button>
+                <button type="reset">RESET</button>
```
Comment (vladiiancu) [inline contact-us.html]: In real life have you ever seen a reset button on any form your used? 
This button can be found only in old tutorials. No actual use for it

**Review by vladiiancu — CHANGES_REQUESTED** [review]

File: contact-us.html (hunk 5 above)
Comment (Alexandru-Bulai) [reply contact-us.html]: True in modern sites you dont see them... i can remove it

File: CSS/contact-us style/form.css (hunk 4 above)
Comment (Alexandru-Bulai) [reply CSS/contact-us style/form.css:68]: Right that line is actually useless calculating that the befault input style has already that values

**Review by vladiiancu — APPROVED** [review]
